### PR TITLE
Fetch rockets

### LINKED ISSUE #13: [4pt] Display rockets - Lists render
- Use `useSelector()` Redux Hook to select the state slices and render lists of rockets in corresponding routes. i.e.:
```javascript
// get rockets data from the store
const rockets = useSelector(state => state.rockets);
```

- You can style the whole application "by hand" or you could use [React Bootstrap](https://react-bootstrap.github.io/), a UI library that could speed up the process. This is a popular library and working with its components would be good practice.
- Render a list of rockets (as per design). For the image of a rocket use the first image in the array of `flickr_images`.

### LINKED ISSUE #17: [4pt]  Fetch rockets - Fetch data
Fetch data from the Rockets endpoint  (https://api.spacexdata.com/v3/rockets) when the application starts (as Rockets is the default view).

Once the data are fetched, dispatch an action to store the selected data in Redux store:

- id
- name
- type
- flickr_images

NOTE: Make sure you only dispatch those actions once and do not add data to store on every re-render (i.e. when changing views / using navigation).

### PR DESCRIPTION
**Pull request by julie-ify**
### In this milestone, I;
 - Fetch data from the Rockets endpoint (https://api.spacexdata.com/v3/rockets) when the application starts 
- Dispatched an action to store the selected data in Redux store
- Dispatched action once and do not add data to store on every re-render (i.e. when changing views / using navigation).
- Used `useSelector()`, Redux Hook, to select the state slices and render lists of rockets in corresponding routes.

### CODE REVIEW

File: src/components/GetRockets.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        item.rocket_name = data[i].rocket_name;
+        item.description = data[i].description;
+        item.flickr_images = data[i].flickr_images;
+        newResponse.push(item);
+      }
+    });
+  return newResponse;
+};
+
+export default getReservation;
```
Comment (ZeenatLawal) [inline src/components/GetRockets.js]: - [ ] I think it'll be good to use .map here rather than using for loop, to follow JS best practices.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @julie-ify , 

Great job so far!:clap: :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

File: src/components/GetRockets.js (hunk 1 above)
Comment (julie-ify) [reply src/components/GetRockets.js]: using map gives me so many errors which I tried fixing but ain't working. But I will give it a try again. Thanks

Comment (julie-ify) [reply src/components/GetRockets.js]: Done!

**Review by ZeenatLawal — APPROVED** [review]
Hi @julie-ify, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

**Pull request merged**